scalafix-core/shared/src/main/scala/scalafix/internal/rule/SingleAbstractMethod.scala

```diff
@@ -0,0 +1,174 @@
+package scalafix.internal.rule
+
+import scala.meta._
+import scalafix.Patch
+import scalafix.SemanticdbIndex
+import scalafix.rule.RuleCtx
+import scalafix.rule.SemanticRule
+
+/*
+http://www.scala-lang.org/files/archive/spec/2.12/06-expressions.html#sam-conversion
+
+SAM conversion
```

Comment:
Is it necessary to inline this comment? A link is enough IMO, this docstring should explain scalafix specific logic.

Reply:
I wanted to do a checklist.